Princesso/Send-IT
# E4131278 add database

This Pull Request adds a database to Send-It and also implements authentication and authorization.

Description:
The following endpoints work on the database
GET api/v1/parcels
GET api/v1/parcels/id
POST api/v1/parcels
PATCH api/v1/parcels/id/destination
PATCH api/v1/id/cancel
GET api/v1/users/id/parcels
POST api/v1/auth/signup
POST api/v1/auth/login

To test these routes 
Ensure you have a database or run the server/database.js and then use postman to test the routes.

Relevant stories
#46 
##E4131278 add database
##4147806  implement authentication and authorization
